docs/theme/assets/site.css

```
@@ -0,0 +1,733 @@
+  padding: 9px 16px;
+  margin: 0;
+  border-bottom: 1px solid var(--line-2);
+  background: none;
+}
+/* Material-style type icon, drawn in the title color via a CSS mask */
+.ld-article .admonition > .admonition-title::before {
+  content: "";
+  width: 14px;
+  height: 14px;
+  flex-shrink: 0;
+  background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)